app/styles/_tables.less

```diff
@@ -151,8 +151,8 @@
   th#severity {width:10px;padding:0;}
   th#reason {width:150px;}
   th#message   {width:100%;} // ensures it gets remaining width
-  th#kind-name,th#message  {
-    .word-break;
```

Comment:
is that true on all the browsers? I'm pretty sure our word-break behaves slightly different in different browsers because it uses the "best" option for each browser